### PR TITLE
website improvement

### PR DESCRIPTION
**Pull request by urvashigupta7**
#723 Website Improvement

## Before
Screenshot from 2020-03-18 19-19-45

## After
Screenshot from 2020-03-18 19-19-22

### On Mobile
Screenshot from 2020-03-18 19-20-51

### CODE REVIEW

**Comment by urvashigupta7** [comment]
@wtrocki Please review. :)

**Comment by wtrocki** [comment]
Thank you so much. Looks good so far. Feel free to improve on top of this and bring some of the quick wins we talked over.

**Comment by urvashigupta7** [comment]
> Thank you so much. Looks good so far. Feel free to improve on top of this and bring some of the quick wins we talked over.

Sure :)

File: website/i18n/en.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "localized-strings": {
     "next": "Next",
     "previous": "Previous",
-    "tagline": "Production ready GraphQL API for your client and server applications",
+    "tagline": "Production ready GraphQL API for your client & server applications",
```
Comment (craicoverflow) [inline website/i18n/en.json]: "and" should be preferrable to "&"

Comment (urvashigupta7) [reply website/i18n/en.json]: Okay @craicoverflow

**Comment by urvashigupta7** [comment]
Screenshot from 2020-03-20 20-24-25
@craicoverflow  I have added the supported databases and init command part. Please review.

**Comment by craicoverflow** [comment]
> @craicoverflow I have added the supported databases and init command part. Please review.

Hi @urvashigupta7, thanks for these great improvements to our website :smile: 

It looks great from a visual perspective. However I think we could do:

The first section is a little bit too wide, it is a bit overbearing visually. It would look much better if the width were reduced to the same with as the navbar and a bit more padding from the top of the navbar, kind of like this:

sc1

I know yourself and @wtrocki had some separate discussions about this so I am missing context, but was the idea to have a GIF instead of a screenshot of the terminal? This would be much more valuable in demonstrating how easy Graphback is to use.

Should the `Supported Databases` section include SQLite @wtrocki ? We do support it.

Also, should the Supported Databases section be above or below the screenshot/GIF as its own section? To me this makes sense but maybe there was a reason for this.

**Comment by wtrocki** [comment]
Fully agree with @craicoverflow on all details. In fact we just pointed out the same issues when chatting directly. Overal is good but some of the padding/reducing margins is needed.

**Comment by wtrocki** [comment]
https://github.com/Kirkhammetz/termly.js was suggested as way to build terminal from code. Layout was build based on our competior

**Comment by wtrocki** [comment]
Worth to mention that we also support REST.

**Comment by urvashigupta7** [comment]
@craicoverflow @wtrocki I will make the corrections as suggested. Should I use gif instead of image?

**Comment by wtrocki** [comment]
I was like - let's put image first and then maybe team can put this js-library. 
Problem with gif is that it reduces page load size and requires deep compression.  IMHO for the moment having image is ok

**Comment by urvashigupta7** [comment]
@craicoverflow why your screenshot and my screenshot has so much difference? 🤔

**Comment by craicoverflow** [comment]
This is a mock of what I think it should look like

On Friday, 20 March 2020, Urvashi Gupta <notifications@github.com> wrote:
> @craicoverflow why your screenshot and my screenshot has so much
difference? 🤔
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or unsubscribe.<
https://ci4.googleusercontent.com/proxy/zSWHrpqwj2J6Z1k_Pi7KnxQyOq-T2lO4krhiNV9sSCuVfUv2O4a_KxnFzGSQmAM3t3jUd2EZGbCVdah3egxAUxHkv63T1I3yfJ4x7RaTZF6EdoviVVtd4NSE4Ypf5NGyYe0GbXJo3l20kF4VgI_W4G_4PrL-kwjw-hLRcbtR324NgwWL1Znf2piLOyDS16Bp18cK9QZWZYAoe8c2EtbndjV3di4jaUBxAWKbHMDVUw=s0-d-e1-ft#https://github.com/notifications/beacon/ACZTDZPHLS3FH6NQEUC3QYLRIOQQ3A5CNFSM4LOSDETKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEPPQCAY.gif
>

-- 

Enda Phelan

Associate Software Engineer

Red Hat <https://www.redhat.com>

Communications House

Cork Road, Waterford, Ireland, X91 NY33

ephelan@redhat.com
<https://www.redhat.com>

**Comment by urvashigupta7** [comment]
Okay @craicoverflow

**Comment by wtrocki** [comment]
As for SQL lite and mysql I'm not sure how much testing we are going to be doing with those.

**Comment by urvashigupta7** [comment]
@craicoverflow @wtrocki I have made the changes as suggested.
## Screenshots
Screenshot from 2020-03-22 09-56-28

Screenshot from 2020-03-22 09-56-41

**Pull request merged**